Documentation/markdown/GitProcess.md

```diff
@@ -0,0 +1,147 @@
+# Git-Process
+
+***
+## GitSetup
+
+### Prerequisites 
+
+Before you can proceed to work with Git, there are several steps, which have to be done before Installation: 
+
+1. Installation of **Git** - find [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+2. Creation of **GitHub account** - find [here](https://github.com/)
+
+### Fork the repository
+
+1. Go to OpenInfraPlatform on [GitHub](https://github.com/tumcms/Open-Infra-Platform).
+
+2. Login to your account and *Fork* OpenInfraPlatform to your account. Now there is https://github.com/<your_username>/Open-Infra-Platform. This is your **origin** repository. 
+
+3. *Clone* your repository to local machine.
+
+	- Open your File Explorer and select the folder, where you want to clone OpenInfraPlatform (e.g. C:\dev). Right mouse click in this folder and select *Git Bash Here*.
+
+	- In the window, which you have opened, in the command line write out:
+	 > git clone https://github.com/<your_username>/Open-Infra-Platform.git
```

Comment:
This can also be copy-pasted directly from github after forking a repository

Reply:
Done